builtin/v9/migration/top.go

```diff
@@ -307,7 +350,317 @@ func MigrateStateTree(ctx context.Context, store cbor.IpldStore, newManifestCID
 
 	elapsed := time.Since(startTime)
 	rate := float64(doneCount) / elapsed.Seconds()
-	log.Log(rt.INFO, "All %d done after %v (%.0f/s). Flushing state tree root.", doneCount, elapsed, rate)
+	log.Log(rt.INFO, "All %d done after %v (%.0f/s). Starting deferred migrations.", doneCount, elapsed, rate)
+
+	// Create the Datacap actor
```

Comment:
For clarity if possible please split into three different migration functions with inputs and outputs.  From talking offline it sounds like this should be possible.

Reply:
Yessir!